### ISSUE TITLE
Cannot run the USA WipEout Pure demo bundled with the Stealth movie on a Japanese PSP

### ISSUE THREAD

**Issue by DrAzathoth** [PSP-Archive/ARK-4#375]
### PSP/Vita Model

PSP-1000 (Fat)

### ARK Version

4.20.69 (DC10) r8

### Homebrew/GAME (if applicable)

Stealth + WipEout Pure [1-4049-9661-3]

### Steps to reproduce

1. Boot up the PSP
2. Insert the UMD (if it is already inserted before turning the PSP on, you will get a disc code error)
3. Make sure your region setting matches the UMD
4. Attempt to run the demo. Instead, it gives you error 80010087.

### USB Charge

Disabled

### Overclock

Disabled

### Power Save

Disabled

### Balanced Energy Mode

Disabled

### Autoboot Launcher

Off

### Disable PSP Go Pause

Off

### Unlock Extra Memory

Disabled

### Memory Stick Speedup

Disabled

### Inferno Cache

Off

### Skip Sony Logos

Off

### Hide PIC0 and PIC1

Off

### Hide MAC Address

Off

### Hide DLC

On

### Turn off LEDs

Disabled

### Verification of form.

- [X] I agree that I properly filled out the items listed above.

**Comment by krazynez** [comment]
Does the Movie portion work?

**Comment by krazynez** [comment]
> > Does the Movie portion work?
> 
> Yes it does. Also, I tried dumping the UMD. I was able to dump the movie partition easily, but no matter what method I tried, I could not dump the game partition.

Yeah that would depend on runlevel. I have hit that issue with other dumping methods also. Does it show up in Custom Launcher?

**Comment by krazynez** [comment]
Under Extra's click Custom Launcher.

If your PSP does not have Extra's then it would be under the Game section.

**Comment by krazynez** [comment]
> > Under Extra's click Custom Launcher.
> > If your PSP does not have Extra's then it would be under the Game section.
> 
> In the Custom Launcher the UMD has no icon and running it just reboots the PSP to the XMB with a disc code error.

Okay yeah that is expected behavior. So the issue is the UMD is marked as UMD_VIDEO instead of UMD_GAME which is why this is happening. I was trying to resolve that before within the CL but it being launched as a Game caused issues. But XMB should not have issues which is an interesting find.

**Comment by krazynez** [comment]
> Yeah, this UMD is a weird one. From what I have heard, no one has ever been able to extract a Redump quality ISO; they only can dump either partition separately and merge them afterwards. It would be nice to finally be able to dump the UMD’s contents properly.
> 
> Also, is the PSP not being able to read a region-locked UMD from another region when booting with it already in the disc drive intended behavior? Even if it is, it would be nice to not have to reinsert the UMD.

I know how to dump a 1:1 :wink:

**Comment by krazynez** [comment]
> > I know how to dump a 1:1 😉
> 
> Really? How?

You need to be running the 1.50 kernel and a dumper that I was working on with another developer.

**Comment by JoseAaronLopezGarcia** [comment]
> Yeah, this UMD is a weird one. From what I have heard, no one has ever been able to extract a Redump quality ISO; they only can dump either partition separately and merge them afterwards. It would be nice to finally be able to dump the UMD’s contents properly.
> 
> Also, is the PSP not being able to read a region-locked UMD from another region when booting with it already in the disc drive intended behavior? Even if it is, it would be nice to not have to reinsert the UMD.

The reason you have to reinsert the disc is because the region change happens at a later stage in the boot process, by that time the system has already tried reading the UMD using the old keys (before being swapped) and thus couldn't read the disk, ejecting and reinserting makes it read it again but by time the keys have been swapped.

**Comment by JoseAaronLopezGarcia** [comment]
You can use DC10 to change UMD region on a more permanent basis to be able to play this disc entirely.
